shinydashboard/lantern/functions/endpoints.R

```diff
@@ -165,17 +166,40 @@ get_fhir_resource_count <- function(fhir_resources_tbl) {
 
 get_capstat_fields_count <- function(capstat_fields_tbl) {
   res <- capstat_fields_tbl %>%
-    group_by(field, exist, fhir_version) %>%
+    group_by(field, exist, fhir_version, extension) %>%
     count() %>%
     filter(exist == "true") %>%
+    filter(extension == "false") %>%
     ungroup() %>%
     select(-exist) %>%
+    select(-extension) %>%
+    rename(Fields = field, Endpoints = n)
+}
+
+get_capstat_extensions_count <- function(capstat_fields_tbl) {
+  res <- capstat_fields_tbl %>%
+    group_by(field, exist, fhir_version, extension) %>%
+    count() %>%
+    filter(exist == "true") %>%
+    filter(extension == "true") %>%
+    ungroup() %>%
+    select(-exist) %>%
+    select(-extension) %>%
```

Comment:
I think the selects here and in `get_capstat_fields_count` can be combined into one select: `select(-exist, -extension)`